assets/bigquery/fishing-events-4-authorization-schema.json

```diff
@@ -0,0 +1,182 @@
+        "mode": "NULLABLE",
+        "fields": [
+            {
+                "name": "eez",
+                "type": "STRING",
+                "mode": "REPEATED",
+                "description": "Exclusive Economic Zone."
+            },
+            {
+                "name": "mpa",
+                "type": "STRING",
+                "mode": "REPEATED",
+                "description": "Maritme private area."
+            },
+            {
+                "name": "rfmo",
+                "type": "STRING",
+                "mode": "REPEATED",
+                "description": "Regional fisheries management organisations."
+            },
+            {
+                "name": "fao",
+                "type": "STRING",
+                "mode": "REPEATED",
+                "description": "Food and Agriculture Organization."
+            },
+            {
+                "name": "major_fao",
+                "type": "STRING",
+                "mode": "REPEATED",
+                "description": "Major FAO."
+            },
+            {
+                "name": "eez12nm",
+                "type": "STRING",
+                "mode": "REPEATED",
+                "description": "EEZ 12nm."
+            },
+            {
+                "name": "high_seas",
+                "type": "STRING",
+                "mode": "REPEATED",
+                "description": "High seas."
+            },
+            {
+                "name": "mpa_no_take_partial",
+                "type": "STRING",
+                "mode": "REPEATED",
+                "description": "MPA no take partial."
+            },
+            {
+                "name": "mpa_no_take",
+                "type": "STRING",
+                "mode": "REPEATED",
+                "description": "MPA no take."
+            }
+        ]
+    },
+    {
+        "name": "start_distance_from_shore_km",
+        "type": "FLOAT",
+        "mode": "NULLABLE",
+        "description": "Distance from shore when event started."
+    },
+    {
+        "name": "end_distance_from_shore_km",
+        "type": "FLOAT",
+        "mode": "NULLABLE",
+        "description": "Distance from shore when event ended."
+    },
+    {
+        "name": "start_distance_from_port_km",
+        "type": "FLOAT",
+        "mode": "NULLABLE",
+        "description": "Distance from port when event started."
+    },
+    {
+        "name": "end_distance_from_port_km",
+        "type": "FLOAT",
+        "mode": "NULLABLE",
+        "description": "Distance from port when event ended."
+    },
+    {
+        "name": "event_info",
+        "type": "STRING",
+        "mode": "NULLABLE",
+        "description": "Info of the event."
+    },
+    {
+        "name": "event_vessels",
+        "type": "STRING",
+        "mode": "NULLABLE",
+        "description": "Event vessels."
+    },
+    {
+        "name": "prod_shiptype",
+        "type": "STRING",
+        "mode": "NULLABLE",
+        "description": "type of ship from prod eyes."
```

Comment:
that looks like a typo or unclear. I'd say: "type of ship as determined for products".